### DISCUSSION TITLE
播放流媒体一段时间莫名闪退

### DISCUSSION THREAD

**Discussion by ShicYang** [hooke007/mpv_PlayKit#438] [mpv 通用]
使用场景是用emby调用mpv-lazy播放nas里的视频，每次播放个十来分钟就会莫名闪退，用了最新版本和EX版本的都是一样，打印了日志也没发现啥内容，不知道从何查起，使用potplayer播放一切正常

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/discussions/379#discussioncomment-7660995

**Reply by ShicYang** [reply]
原来一直有这个问题啊，我试试这个方法
